views/join.haml

```diff
@@ -0,0 +1,57 @@
+!!!
+%html{:lang => 'en'}
+  %head
```

Comment:
Any reason not use the `layout`?

Reply:
Depened on design file,login and register page not incloud item such as `Home` `Document` `Dashboard`,so,I code those page without use layout file.Maybe I don’t understand enough about `layout`. If you have a good suggestion,I will fix it.